app/queries/content_changes_for_email_query.rb

```diff
@@ -1,9 +1,7 @@
 class ContentChangesForEmailQuery
   def self.call(email)
     subscription_content = SubscriptionContent
-      .where("subscription_contents.content_change_id = content_changes.id")
-      .where(email: email)
-      .exists
-    ContentChange.where(subscription_content)
+      .where(email: email).select(:content_change_id)
+    ContentChange.where(id: subscription_content)
```

Comment:
Perhaps inline in the SubscriptionContent part of the query? Or, update the variable name to make sense, now that it's some `content_change_id`s.